src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -101,5 +102,27 @@ public async Task<IActionResult> GetApprenticeshipsFilterValues([FromQuery]GetAp
 
             return Ok(response);
         }
+
+        [HttpPost]
+        [Route("{apprenticeshipId}/stop-apprenticeship")]
+        public async Task<IActionResult> StopApprenticeship(long apprenticeshipId, [FromBody] StopApprenticeshipRequest request)
+        {
+            try
+            {
+                await _mediator.Send(new StopApprenticeshipCommand(
+                    request.AccountId,
+                    apprenticeshipId,
+                    request.StopDate,
+                    request.MadeRedundant,
+                    request.UserInfo));
+
+                return Ok();
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, e.Message);
+                return BadRequest(e.Message);
```

Comment:
I'm not sure it's right to always return a `badrequest` response type here - the request might have been valid, but something else might have gone awry.

Reply:
I've taken this out, realised that the catch in the action was suppressing the `DomainException` error messages from the Api